about/glossary.rst

```diff
@@ -22,13 +22,35 @@ Glossary
       Amazon Web Services: Cloud computing infrastructure provided by Amazon 
       (United States).   
    
+   BRDF
+      “Bidirectional reflectance distribution function” is a theoretical concept that describes the 
+      relationship between light and an opaque surface, using a target's irradiance geometry and 
+      the remote sensing system’s relative angle to the target.
+   
+   C2
+      Collection 2 - Digital Earth Australia's second collection of Landsat data that has now 
+      been superceded by Collection 3 (C3).
+      
+   C3
+      Collection 3 - The third collection of Digital Earth Australia's Landsat or Sentinel data 
+      and the most up-to-date collection available.
+   
    CEOS-SEO
       Committee on Earth Observation, Systems Engineering Office
       http://ceos-cube.org/
       
    COG
       Cloud Optimised GeoTIFF, a data file format optimised for efficient 
       workflows on the cloud and remote reading. 
+  
+   Collection
+     C1, C2, C3 ...n - the whole suite of data from the Landsat or Sentinel mission. 
+     Collections are updated when there are fundamental changes and upgrades to the data suite that 
+     make it incompatible with the existing collection. Therefore a collection upgrade is more 
+     akin to a movie franchise reboot than a re-release (think of C2 as the 1968 version of Planet 
+     of the Apes starring Charlton Heston alongside actors playing apes in hairy costume, and 
+     C3 as the 2011 reboot, Rise of the Planet of the Apes, where Andy Serkis played the CGI 
+     ape Caesar using of ‘performance capture’ acting).
```

Comment:
Loving these descriptions 😀. Small typo:
> played the CGI ape Caesar using ***of*** ‘performance capture’ acting

Not having seen the films in question I might be biased, but I don't think a lot is added by the more specific description (maybe that could go in a footnote). Ending at `more akin to a movie franchise reboot than a re-release` suffices imo.

Reply:
Removed the extra commentary as suggested.